index.js

```diff
@@ -48,9 +48,13 @@ Guard.prototype = {
       }
 
       if (!Array.isArray(permissions)) {
-        return next(new UnauthorizedError('permissions_invalid', {
-          message: 'Permissions should be an Array. Bad format?'
-        }))
+        if (typeof permissions === 'string') {
```

Comment:
Yes please :) I've also merged your PR to remedy the failing Travis builds so when CI is green we're good to merge and release 🤘